iothub/device/src/Common/ErrorCode.cs

```diff
@@ -11,6 +11,7 @@ namespace Microsoft.Azure.Devices.Client.Errors
     /// </remarks>
     public enum ErrorCode
     {
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
```

Comment:
I could not find a way to add this to the global file so had to do it in place. Also if anyone has good context on any of these errors that would be good to include as comments, let me know.